<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget{
background-color: #1B262C;
}

QLineEdit{
background-color: rgb(18, 26, 30);
color: white;
border: 2px solid gray;
border-radius: 5px;}

QLineEdit:hover{
border: 2px solid rgb(174, 173, 172);
}

QLineEdit:focus{
border: 2px solid rgb(7,134,193)}
</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>451</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="startBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
 background-color: #3282B8;
 color: rgb(228,240,237);
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="supportedBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
 background-color: #3282B8;
 color: rgb(228,240,237);
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>View Supported Languages</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="settingsBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
 background-color: #3282B8;
 color: rgb(228,240,237);
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitBtn">
       <property name="styleSheet">
        <string notr="true">QPushButton{
 background-color: #3282B8;
 color: rgb(228,240,237);
 border-style: outset;
 border-width: 2px;
 border-radius: 10px;
 border-color: black;
 min-width: 10em;
 padding: 6px;}

QPushButton:hover{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:focus{
border-color: lightgray;
background-color: rgb(60, 158, 223);
}

QPushButton:pressed {
 background-color: #0F4C75;
 border-style: inset;
}</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
